desktop/storeService.ts

```diff
@@ -1,11 +1,26 @@
-const Store = require('electron-store');
+import { nativeTheme } from 'electron';
+import Store from 'electron-store';
+import { TypedSchemaStore } from '../shared/types';
 
 class StoreService {
-  static store: typeof Store;
+  public static store: Store<TypedSchemaStore>;
```

Comment:
Can we get rid of `init` and just move construction of the Store here?
```
public static store = new Store<TypedSchemaStore>({ ... });
```

Reply:
I think it's ideal to have the class with a constructor function. So we can always check for undefined in the store.

Reply:
Why do we need to check for undefined if we can just get rid of possibility of undefined?

Reply:
store should be singleton for easy import and usage in other classes